### PR TITLE
Unmark the file as cloud_available before size check

### PR DESCRIPTION
**Pull request by wengole**
This ensures replacement files that are smaller than the current threshold will be served from Plone and override what's in S3

### CODE REVIEW

**Comment by wengole** [comment]
@mattss For review...

File: src/netsight/cloudstorage/adapters.py [hunk 1]
```diff
@@ -245,6 +245,10 @@ def enqueue(self, enforce_file_size=True):
                 # Ignore empty fields
                 continue
 
+            # Remove existing cloud info, assuming file data has changed
+            if field['name'] in cloud_available:
+                del cloud_available[field['name']]
+
             min_size = get_value_from_registry('min_file_size')
             # TODO: Move this out for bulk uploading, not manual
```
Comment (mattss) [inline src/netsight/cloudstorage/adapters.py]: I don't think we need this comment any more - do you?

**Comment by wengole** [comment]
@mattss All done?

**Comment by mattss** [comment]
Love it

**Pull request merged**